src/Libraries/RevitNodes/Elements/FamilyInstance.cs

```diff
@@ -275,6 +329,12 @@ private void InternalSetLevel(Autodesk.Revit.DB.Level level)
             TransactionManager.Instance.TransactionTaskDone();
         }
 
+        private void InternalSetView(Autodesk.Revit.DB.View view)
+        {
+            if (InternalFamilyInstance.OwnerViewId.Compare(view.Id) == 0)
+                return;
```

Comment:
Why do we need this? It looks this method always does nothing?

Reply:
I was following what was already in there for family instances. But, as I said below, I am no longer interested in contributing this pull request to DynamoRevit. I will just add it to Rhythm and wait for you to copy the code in 2 years.